package.json

```diff
@@ -25,17 +25,18 @@
   },
   "homepage": "https://github.com/blackbaud/skyux-sdk-builder#readme",
   "peerDependencies": {
-    "@angular/compiler-cli": "^7.2.1",
-    "@angular/compiler": "^7.2.1",
-    "@angular/common": "^7.2.1",
-    "@angular/core": "^7.2.1",
-    "@angular/forms": "^7.2.1",
-    "@angular/http": "^7.2.1",
-    "@angular/platform-browser": "^7.2.1",
-    "@angular/platform-browser-dynamic": "^7.2.1",
-    "@angular/router": "^7.2.1",
+    "@angular/compiler-cli": "^7.2.4",
```

Comment:
Are we using a new feature of Angular 7.2.4 so that we need to force consumers to update?  If not, my vote would be to keep the old peer dependency version.  Same goes for the rest of the peer dependencies.

Reply:
Good point! I'll adjust.